pandas/io/pytables.py

```diff
@@ -4242,38 +4243,29 @@ def write_data_chunk(self, rows, indexes, mask, values):
             if not np.prod(v.shape):
                 return
 
-        try:
```

Comment:
So just nothing hitting these in the test cases right? If so, any insight from git blame as to why this was here in the first place?

Reply:
Pretty much, yah.  The blame for these goes to back before we were using github AFAICT, so there arent associated PRs/threads.